### PR TITLE
FIX (#124): Added seq2seq model for translation in Model Zoo

### PR DESCRIPTION
**Pull request by AtharvaKhare**
The neural network translates English to French. The model is
taken from https://github.com/fchollet/keras/blob/master/examples/lstm_seq2seq.py
Changed `caffe_tensorflow_keras_install.sh` to install latest
version of keras.
Fixes https://github.com/Cloud-CV/Fabrik/issues/124
Note: Keras version 2.0.8 ~~and Tensorflow version 1.3.0~~ is needed
for this model to work.

### CODE REVIEW

**Comment by AtharvaKhare** [comment]
Seems like I cant use 2.0.8 (https://github.com/Cloud-CV/Fabrik/pull/104). Searching for workaround.

File: ide/static/js/topBar.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ class TopBar extends React.Component {
                       <li><ModelElement importNet={this.props.importNet} framework="caffe" id="All_CNN">All CNN</ModelElement></li>
                       <li><ModelElement importNet={this.props.importNet} framework="caffe" id="Squeezenet">Squeezenet</ModelElement></li>
                       <li><ModelElement importNet={this.props.importNet} framework="keras" id="textGeneration">Text Generation</ModelElement></li>
+                      <li><ModelElement importNet={this.props.importNet} framework="keras" id="seq2seq_lang">Seq2seq Translation</ModelElement></li>
```
Comment (utsavgarg) [inline ide/static/js/topBar.js]: Seq2Seq Translation

File: requirements/common.txt [hunk 2]
```diff
@@ -2,3 +2,5 @@ Django==1.9.3
 pyaml==15.8.2
 django-allauth==0.30.0
 psycopg2==2.7.3
+scikit-image==0.13.1
+matplotlib==2.1.0
```
Comment (utsavgarg) [inline requirements/common.txt]: Aren't these already installed in `caffe_tensorflow_keras_install.sh`?

Comment (AtharvaKhare) [reply requirements/common.txt]: Matplotlib isnt in that script. scikit-image does not get installed through the script, idk why, but will remove it if you insist.

**Comment by AtharvaKhare** [comment]
@utsavgarg Any more changes?

File: requirements/common.txt (hunk 2 above)
Comment (utsavgarg) [reply requirements/common.txt]: The travis build was fine before as well so `scikit-image` must have been getting installed through the script, and `matplotlib` is not there separately as its a dependency for `scikit-image`  and is installed automatically.

Comment (AtharvaKhare) [reply requirements/common.txt]: I tried installing twice on different VMs, and faced the issues. Even @manuhortet faced this issue. (Need for installing scikit-image and matplotlib)

Anyways, removed the two lines.

**Comment by utsavgarg** [comment]
Nice work. :)

**Pull request merged**